src/index.html

```diff
@@ -2,16 +2,104 @@
 <html lang="en">
   <head>
     <meta charset="UTF-8">
-    <meta
-      name="viewport"
+    <meta name="viewport"
       content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
     >
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>HTML Form</title>
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application" method="post">
+      <fieldset class="fieldset">
+        <legend>Personal information</legend>
+        <label class="form-field">
+          Surname:
+          <input type="text" name="surname" autocomplete="off">
+        </label>
```

Comment:
Fix all places
![2023-02-09_14h20_45](https://user-images.githubusercontent.com/61904995/217811432-75efaed2-7f86-48d1-8933-e6dbec1624d3.png)
```suggestion
          <input
            type="text"
            name="surname" 
            autocomplete="off"
          >
        </label>
```